tests/general/InvalidLinkBearTest.py

```diff
@@ -128,7 +128,11 @@ def test_run(self):
         http://httpbin.org/status/503
         http://www.notexample.com/404
         http://exampe.com/404
-        http://example.co.in/404"""
+        http://example.co.in/404
+        <ruleset name="test" xmlns="http://this.isa.namespace/ruleset/7.0.0"
+        xmlns:xsi="http://this.is.another/kindof/namespace"
+        xsi:schemaLocation="http://this.namespace.dosent/exists/7.0.0"
```

Comment:
If that's the case, I guess we should create a separate issue for that because none of the tests are written yet to check the severity.